packages/core/src/amazonqGumby/chat/session/session.ts

```diff
@@ -10,7 +10,8 @@ export enum ConversationState {
     PROMPT_JAVA_HOME,
     COMPILING,
     JOB_SUBMITTED,
-    WAITING_FOR_INPUT,
+    WAITING_FOR_HIL_INPUT,
+    WAITING_FOR_OBJECTIVE,
```

Comment:
```suggestion
    WAITING_FOR_TRANSFORMATION_OBJECTIVE,
```
To match all other references of transformation objective used above